{"id": "5340", "title": "Bug 5340: Cancelling add exception breakpoint has no effect", "description": "", "OB": "When adding an exception breakpoint by clicking the J! button in the breakpoints pane of the debug perspective, the progress dialog window immediately closes. However, the exception list still appears even though the cancellation request was made. Currently, the cancel button does not disable when calling ProgressMonitorDialog.run, even though it should. To fix this issue, the cancellation request should be honored or the cancel button should be disabled (by passing false for the cancellable parameter when calling ProgressMonitorDialog.run).", "EB": "", "SR": "1) Go to the breakpoints pane in the debug perspective. 2) Click the J! button to add an exception. 3) In the progress dialog window, click cancel."}